src/GraphQL.DataLoader.Tests/Types/QueryType.cs

```diff
@@ -79,10 +79,7 @@ public QueryType(IDataLoaderContextAccessor accessor, IUsersStore users, IOrders
                     users.GetUsersByIdAsync);
 
                 var ids = ctx.GetArgument<IEnumerable<int>>("ids");
-                // note: does not work properly without ToList, because LoadAsync would not have
-                // been called, so the ids would not have been queued for execution prior to the
-                // first call to GetResultAsync
-                var ret = ids.Select(id => loader.LoadAsync(id)).ToList();
+                var ret = ids.Select(id => loader.LoadAsync(id));
```

Comment:
This test failed before this patch, as it was known to do at the time I wrote it.  This actually a perfect sample of the bad design, requiring users to add `ToList` when such behavior is required.